### PR TITLE
add capitalize bytes.center + fix str.center

### PR DESCRIPTION
**Pull request by jgirardet**
Add bytes.center
fix str.center
add some tests

introduce `is_byt`e to get value of a byte object : byte or byterray.
I'cant get out of a clone() in this function. maybe you have an idea ?

Should follow in another  PR : is_bytes_like : which basically would do the same thing, including memory view.

also added a get_value() to PyBytes as convenient way to deal with PyBytes Value in other function. Id'l like it be also added to byterray when refactoring it with PyByteInner.

### CODE REVIEW

File: vm/src/obj/rien.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+use crate::obj:
```
Comment (windelbouwman) [inline vm/src/obj/rien.rs]: What is the purpose of this file?

**Comment by windelbouwman** [comment]
You added two empty files, `Pipfile` and `tests/snippets/resources/__init__.py`, what are they for?

File: vm/src/obj/rien.rs (hunk 1 above)
Comment (jgirardet) [reply vm/src/obj/rien.rs]: Oh sorry i didn't see all the mess in my commit. I 'll fix it tomorrow

**Comment by jgirardet** [comment]
Ok that"s fixed.

**Comment by windelbouwman** [comment]
Looks good to me! We might be able to share some code between `str`, `bytes` and `bytearray`, but I leave that for a later opportunity.

File: vm/src/obj/objbyteinner.rs [hunk 2]
```diff
@@ -346,4 +360,39 @@ impl PyByteInner {
...
+        // merge all
+        let mut res = vec![fillbyte; ln];
+        res.extend_from_slice(&self.elements[..]);
+        res.extend_from_slice(&vec![fillbyte; rn][..]);
+
+        res
+    }
+}
+
+pub fn is_byte(obj: &PyObjectRef) -> Option<Vec<u8>> {
```
Comment (adrian17) [inline vm/src/obj/objbyteinner.rs]: could this return a ref to the Vec instead of copying it?

**Pull request merged**